artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ActiveMQServerImpl.java

```diff
@@ -477,12 +484,44 @@ public final synchronized void start() throws Exception {
       }
    }
 
+   @Override
+   public CriticalAnalyzer getCriticalAnalyzer() {
+      return this.analyzer;
+   }
+
    private void internalStart() throws Exception {
       if (state != SERVER_STATE.STOPPED) {
          logger.debug("Server already started!");
          return;
       }
 
+      if (!configuration.isAnalyzeCritical()) {
```

Comment:
I thought the configuration was being parsed inside ActiveMQServerImpl. (Bad memory, I forget about stuff months after working on stuff :) )... I will move it to constructor.